Editor/Scripts/wind_forces_tutorial.py

```diff
@@ -0,0 +1,94 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import sys
+import azlmbr
+
+from PySide2.QtWidgets import QMenuBar
```

Comment:
Need to import QtWidgets to find UI elements to highlight by pattern.

```suggestion
from PySide2 import QtWidgets
from PySide2.QtWidgets import QMenuBar
```